app/src/main/java/org/luxoft/sdl_core/SdlLauncherService.java

```diff
@@ -106,17 +118,44 @@ private void showToastMessage(String message) {
         toast.show();
     }
 
+    private void updateServiceNotification(String text) {
+        Notification notification = getServiceNotification(text);
+
+        NotificationManager notificationManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        notificationManager.notify(FOREGROUND_SERVICE_ID, notification);
+    }
+
+
```

Comment:
redundant empty line

Reply:
@VadymLuchko fixed in https://github.com/CustomSDL/sdl_android_launcher/pull/5/commits/a71426ea4cf4330474712b02a694fff1a2356020